src/promptflow-evals/promptflow/evals/evaluate/_evaluate.py

```diff
@@ -283,21 +284,64 @@ def evaluate(
     :keyword data: Path to the data to be evaluated or passed to target if target is set.
         Only .jsonl format files are supported.  `target` and `data` both cannot be None
     :paramtype data: Optional[str]
-    :keyword evaluator_config: Configuration for evaluators.
+    :keyword evaluators: Evaluators to be used for evaluation. It should be a dictionary with key as alias for evaluator
+        and value as the evaluator function.
+    :paramtype evaluators: Optional[Dict[str, Callable]
+    :keyword evaluator_config: Configuration for evaluators. The configuration should be a dictionary with evaluator
+        names as keys and a dictionary of column mappings as values. The column mappings should be a dictionary with
+        keys as the column names in the evaluator input and values as the column names in the input data or data
+        generated by target.
     :paramtype evaluator_config: Optional[Dict[str, Dict[str, str]]
-    :keyword output_path: The local folder path to save evaluation artifacts to if set
+    :keyword output_path: The local folder or file path to save evaluation results to if set. If folder path is provided
+          the results will be saved to a file named `evaluation_results.json` in the folder.
     :paramtype output_path: Optional[str]
-    :keyword azure_ai_project: Logs evaluation results to AI Studio
+    :keyword azure_ai_project: Logs evaluation results to AI Studio if set.
     :paramtype azure_ai_project: Optional[Dict]
-    :return: A EvaluationResult object.
-    :rtype: ~azure.ai.generative.evaluate.EvaluationResult
+    :return: Evaluation results.
+    :rtype: dict
+
+    :Example:
+
+    Evaluate API can be used as follows:
+
+    .. code-block:: python
+
+            from promptflow.evals.evaluate import evaluate
+            from promptflow.evals.evaluators import RelevanceEvaluator, CohereEvaluator
+
+            coherence_eval = CohereEvaluator()

```

Comment:
Add ... to the constructor?
coherence_eval = CohereEvaluator(...)

Reply:
Thanks for catching that. Have fixed it